### PR TITLE
#39: Added first round of documentation updates.

### PR DESCRIPTION
**Pull request by Decipher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+The project has a defined set of default configuration attributes so it will work without any custom configuration. All customisation comes in the form of overrides as undefined attributes will revert to the default.
+
+[![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox) [![Documentation Status](https://readthedocs.org/projects/beetbox/badge/?version=latest)](http://beetbox.readthedocs.org/en/latest/?badge=latest)
+
+
+## Features
+
+* Supports for different VMs per branch.
+* Takes advantage of Vagrant 1.8’s snapshots to manage VMs for speed and disk efficiency.
+* Uses ansible for provision but it's not required on the host machine.
```
Comment (thom8) [inline README.md]: Support for ..
Takes advantage of Vagrant 1.8’s linked clones support ..
Uses ansible 2 for provisioning .. 

or should we just remove this for now?

Comment (Decipher) [inline README.md]: Partial bad copy paste of a suggested edit. Will make the changes now and force push.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,51 @@
-# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
-
-## Overview
+# Beetbox - Drupal development on Vagrant
```
Comment (thom8) [inline README.md:1]: Are we going to change this to L*MP development on Vagrant?

Comment (Decipher) [inline README.md:1]: I figured that change was still pending, but we can go ahead with that change prior to beta 1. Really only wanted to push the approved changes although I did sneak the **Features** change in which is still pending.

**Pull request merged**